bpfd/src/storage.rs

```diff
@@ -203,4 +206,25 @@ impl StorageManager {
             info!("Shutdown CSI Plugin {}", addr);
         });
     }
+
+    #[allow(dead_code)] // TODO: Remove this when the storage manager is fully implemented
+    fn create_bpffs(&self, path: &Path) -> anyhow::Result<()> {
+        create_bpffs(
+            path.as_os_str()
+                .to_str()
+                .expect("unable to convert path to str"),
+        )
+    }
+
+    #[allow(dead_code)] // TODO: Remove this when the storage manager is fully implemented
+    fn pin_map_to_bpffs(
```

Comment:
So for duping a map to a new bpffs we'll just do an aya `from_pin` call to get the mapdata and then pin again to the new bpffs right?

Reply:
Yep. The whole process is:

- MapData::from_pin() - which behind the scenes does BPF_OBJ_GET from $basePath
- Then MapData::pin() - which does BPF_OBJ_PIN to $targetPath

On my TODO list is to check whether we can `cp` the pinned object from one bpffs to another.
That's riskier, so we'll stick with the above for now.

Reply:
Sounds good to me Thanks!